bindings/nodejs/examples/how_tos/sign_and_verify_ed25519/sign-ed25519.ts

```diff
@@ -62,11 +62,11 @@ async function run() {
             bip44Chain,
         );
         console.log(
-            `Public key: ${ed25519Signature.publicKey}\nSignature: ${ed25519Signature.signature}`,
+            `Public key: ${ed25519Signature.publicKey.publicKey}\nSignature: ${ed25519Signature.signature}`,
```

Comment:
Maybe we can add a method like `toHexString`  or something to avoid this duplication and give a hint about the type?

Reply:
> and give a hint about the type
What do you  mean with this?

Reply:
only that the retuned type is string pretty much.

Reply:
Sounds stupid to me to add a method just to have a different name
Anyone else have an opinion on this?

Reply:
yeah, it's not the nicest thing, but `publicKey.publicKey` is also not great :D

Reply:
I don't mind it much though.

Reply:
Accessor method?

Reply:
We removed them for the other types since the field is public

Reply:
`publicKey()` is not allowed, because the field is already named like this, added `ed25519PublicKey()` instead